frontend/app/routes/protected/renew/$id/member-selection.tsx

```diff
@@ -0,0 +1,195 @@
+import type { ReactNode } from 'react';
+
+import type { ActionFunctionArgs, LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
+import { redirect } from '@remix-run/node';
+import { useFetcher, useLoaderData, useParams } from '@remix-run/react';
+
+import { faChevronLeft, faChevronRight, faCircleCheck } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useTranslation } from 'react-i18next';
+
+import { TYPES } from '~/.server/constants';
+import { getProtectedChildrenState, loadProtectedRenewState } from '~/.server/routes/helpers/protected-renew-route-helpers';
+import type { AppLinkProps } from '~/components/app-link';
+import { AppLink } from '~/components/app-link';
+import { Button, ButtonLink } from '~/components/buttons';
+import { LoadingButton } from '~/components/loading-button';
+import { Progress } from '~/components/progress';
+import { pageIds } from '~/page-ids';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT } from '~/utils/locale-utils.server';
+import { getLogger } from '~/utils/logging.server';
+import { mergeMeta } from '~/utils/meta-utils';
+import type { RouteHandleData } from '~/utils/route-utils';
+import { getPathById } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
+
+enum FormAction {
+  Add = 'add',
+  Continue = 'continue',
+  Cancel = 'cancel',
+  Save = 'save',
+  Remove = 'remove',
+  Back = 'back',
+}
```

Comment:
We can remove `Add` and `Remove` as they aren't being used.

Reply:
Removed the unused enums